samples/vxflex_v260_ocp_410.yaml

```diff
@@ -38,23 +38,15 @@ spec:
       # Leave as blank to use all nodes
       # Allowed values: map of key-value pairs
       # Default value: None
-      # Examples:
-      #  node-role.kubernetes.io/control-plane: ""
       nodeSelector:
-      # Uncomment if nodes you wish to use have the node-role.kubernetes.io/master taint
-      #  node-role.kubernetes.io/master: ""
```

Comment:
Please retain Examples of master under nodeSelector and tolerations for OCP 4.10 and K8s 1.23 samples

Reply:
done